lib/datasource.js

```diff
@@ -992,7 +1004,9 @@ DataSource.prototype.discoverModelProperties = function(modelName, options, cb)
   cb = cb || utils.createPromiseCallback();
 
   if (this.connector.discoverModelProperties) {
-    this.connector.discoverModelProperties(modelName, options, cb);
+    var nameMapper = options.nameMapper || this.mapName;
+    var name = nameMapper('table', modelName);
+    this.connector.discoverModelProperties(name, options, cb);
```

Comment:
function `fromDBName` in `mapName` will change the value of `modelName` here, and same for all discover apis.
e.g. the postgresql fails due to `fromDBName` convert `customer_test` to `CustomerTest` 
cc @loay can confirm with other connectors.